nav_msgs/Odometry odom_gt
sensor_msgs/Image seg_img
sensor_msgs/Imu imu
sensor_msgs/PointCloud2 lidar
